src/python/pants/backend/python/tasks/pex_build_util.py

```diff
@@ -158,3 +164,28 @@ def _resolve_multi(interpreter, requirements, platforms, find_links):
       allow_prereleases=python_setup.resolver_allow_prereleases)
 
   return distributions
+
+
+def build_req_libs_provided_by_setup_file(context, local_built_dists, class_name):
+  """Build a requirements library from a local wheel.
```

Comment:
the use of the term 'build' here is misleading.. how about `inject_req_libs...`?

Reply:
also, how about taking just a `build_graph` vs the entire `context`?

Reply:
✅